app/utils/extended-theme.ts

```diff
@@ -99,9 +99,24 @@ export const extendedTheme = {
 			from: { height: 'var(--radix-accordion-content-height)' },
 			to: { height: '0' },
 		},
+		'roll-reveal': {
```

Comment:
We want to make it easy for people to remove unused stuff if they want to. Having this code isn't really all that harmful, but I think it would be best if we add a comment here about what they can delete or even better if we can put these animation keyframes in the marketing index page and import them here. That way when people delete that page, TypeScript can tell them to delete the reference here. Perhaps we only need to have the marketing index page export the keyframes names and this imports those names for use here. That way there's a strong relationship between those keyframes and where they're used. We just want to make sure it's easy for people to know what to remove when they bootstrap a new Epic app.

To be clear, you don't have to do the same in the workshop. It's fine to keep it the way you have it for the workshop.